### PR TITLE
Drafted 3 Core entities

### LINKED ISSUE #3: Draft core entities
* Visitor
* Activity
* Leader

### PR DESCRIPTION
**Pull request by Lehersun**
The test for Visitor is mixed in GroupLife\Core namespace.
Discribed Activity & Leader entities.
Added strict type for Visitor class functions

### CODE REVIEW

File: src/Activity.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace GroupLife\Core;
+
+class Activity
+{
+    private $name;
+
+    public function __construct(string $name)
+    {
+    $this->name = $name;
```
Comment (Magomogo) [inline src/Activity.php]: Please fix the code style

File: src/Visitor.php [hunk 2]
```diff
@@ -1,18 +1,19 @@
 <?php
 
-namespace Core;
+namespace GroupLife\Core;
+
```
Comment (Magomogo) [inline src/Visitor.php:6]: Hmmm, I already saw this change in different pull request, something is wrong here.

**Comment by Magomogo** [comment]
It's good to see that you using strict types. 

What do you think if we go even stricter, by enabling the [strict mode](https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.strict)?

**Review by Magomogo — CHANGES_REQUESTED** [review]

File: src/Visitor.php (hunk 2 above)
Comment (Lehersun) [reply src/Visitor.php:6]: still have problems with what to take and where to get it from

Comment (Magomogo) [reply src/Visitor.php:6]: If you did come change in different PR, is it not necessary to repeat it. The change will propagate eventually when PR will be accepted.

**Review by Magomogo — APPROVED** [review]

**Comment by Magomogo** [comment]
Please fix the build. You may run `composer test` to see the errors.

**Pull request merged**